src/Round.js

```diff
@@ -9,29 +9,19 @@ class Round {
     returnCurrentCard = (card) => {
       return this.deck[0];
     }
-    takeTurn = (guess) => {
-      let newTurn = new Turn();
+    takeTurn = (guess) => {     
       let currentCard = this.returnCurrentCard();
-      return this.getResult(guess, newTurn, currentCard);
-    }
-    getResult = (guess, newTurn, currentCard) =>{
-      return this.evaluateGuess(guess, newTurn, currentCard)
-    }
-    evaluateGuess = (guess, newTurn, currentCard) =>{
-      let result = newTurn.evaluateGuess(guess, currentCard);
-      this.handleResult(result, currentCard)
-      return newTurn.giveFeedback(result);
-    }
-    handleResult = (result, currentCard) =>{
-      !result ? this.incorrectGuesses.unshift(currentCard.id) : null;
+      let newTurn = new Turn(guess, currentCard);
+      !newTurn.evaluateGuess() ? this.incorrectGuesses.unshift(currentCard.id) : null;
       this.deck.shift();
       this.turns++;
+      return newTurn.giveFeedback();
```

Comment:
I am unsure if this is considered SRP, but the amount of methods was too many to be deemed best practice.